docs/css/guides.css

```diff
@@ -32,6 +33,9 @@
 
 .guides-post-row {
     width: 100%;
+    background-color: #f7f7f7;
+    padding: 20px;
```

Comment:
My preference would be more padding/margin round the edge, to make it look more consistent with blogs.